packages/models-library/src/models_library/boot_options.py

```diff
@@ -0,0 +1,69 @@
+from typing import Dict, Optional
+
+from pydantic import BaseModel, constr, root_validator
+from typing_extensions import TypedDict
+
+ENV_VAR_KEY_RE = r"[a-zA-Z][a-azA-Z0-9_]*"
+EnvVarKey = constr(regex=ENV_VAR_KEY_RE)
+
+
+class BootOptionItem(TypedDict):
+    label: str
```

Comment:
question, how are these boot options used? is that always as an ENV variable?
if yes you could add some validators to ensure they follow ENV conventions (no space, no -, etc...)